### PR TITLE
Add perks and benefits block

### LINKED ISSUE #482: [About Us] PerksAndBenefits

### PR DESCRIPTION
**Pull request by mdubus**
# Pull Request

## Related issue
Closes #482

## What does this PR do?
- Addition of the new block "Perks & benefits"
- Addition of the matching story inside Storybook
- Addition of a new hook `useWindowSize` to get the window's size
- Addition of a new utility `isBreakpointSmaller` to check if a window size is smaller than a breakpoint

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/hooks/useWindowSize.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect } from 'react'
+
+const useWindowSize = () => {
+  const [windowSize, setWindowSize] = useState({
+    width: undefined,
+    height: undefined,
+  })
```
Comment (bidoubiwa) [inline src/hooks/useWindowSize.js:7]:
```suggestion
  })
  
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
the worst review ever 😂

File: src/hooks/useWindowSize.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    const handleResize = () => {
+      setWindowSize({
+        width: window.innerWidth,
+        height: window.innerHeight,
+      })
+    }
+    window.addEventListener('resize', handleResize)
+    handleResize()
+    return () => window.removeEventListener('resize', handleResize)
+  }, [])
```
Comment (bidoubiwa) [inline src/hooks/useWindowSize.js:18]:
```suggestion
  }, [])
  
```

**Comment by mdubus** [comment]
bors merge

**Pull request merged**